main.go

```diff
@@ -395,6 +405,66 @@ func NewExporter(server string, timeout time.Duration, logger log.Logger) *Expor
 			[]string{"slab"},
 			nil,
 		),
+		itemsNumberHot: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "items_number_hot"),
+			"Number of items presently stored in the HOT LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsNumberWarm: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "items_number_warm"),
+			"Number of items presently stored in the WARM LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsNumberCold: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "items_number_cold"),
+			"Number of items presently stored in the COLD LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsNumberTemp: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "items_number_temp"),
+			"Number of items presently stored in the TEMPORARY LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsAgeOldesHot: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "items_age_hot"),
```

Comment:
Looking at the memcached code, this seems like it's in seconds. So the unit here would be in seconds.

```suggestion
			prometheus.BuildFQName(namespace, subsystemSlab, "hot_age_seconds"),
```

Reply:
ack!